docs/features/multiple-networks/multi-homing.md

```diff
@@ -1,33 +1,99 @@
+their IP addresses, which is often not possible with the primary network.
+
+In OVN-K, multihoming supports several virt-specific features. These
+include [persistent IP addresses for virtualization](#persistent-ip-addresses-for-virtualization-workloads) workloads,
+ensuring that VMs retain their IP addresses even when they move across nodes. This enhances workload mobility and
+minimizes disruptions.
+
+Multihoming is also compatible with the multi-network policy API, which can provide further security rules on the
+traffic.
 
-Below you will find example attachment configurations for each of the current
-topologies OVN-K allows for secondary networks.
+## User-Stories
+
+- As a Cluster-Admin, I want to configure secondary networks for specific pods so that I can enable
+  specialized/sensitive workloads with distinct network requirements.
+- As a Cluster-Admin, I want to facilitate seamless live migration of VMs within so that I can maintain established TCP
+  connections and preserve VM IP configurations during migration.
+
+## User-Cases
+
+- cluster-wide overlay network on layer 2:
+  In this case example, two VMs from different namespaces - VMA and VMC - are connected over a secondary-network.
+  VMB is not exposed to this traffic.
+  ![multi-homing-use-case-layer2](multi-homing-use-case-layer2.png)
+- cluster-wide overlay network on localnet:
+  In this case example, Pod and VM workloads accessing a relational DB reachable via the physical network (i.e. deployed
+  outside Kubernetes).
+  ![multi-homing-use-case-localnet](multi-homing-use-case-localnet.png)
+
+## How to enable this feature on an OVN-Kubernetes cluster?
+
+This feature is enabled by default on all OVN-Kubernetes clusters. You don't need to do anything extra to start using
+this feature. There is a Feature Config option `enable-multi-network` under `OVNKubernetesFeatureConfig` config that can
+be used to disable this feature.
+
+## Workflow Description
+
+After a pod is scheduled on a particular Kubernetes node, kubelet will invoke
+the meta-plugin installed on the cluster (such as [Multus](https://github.com/k8snetworkplumbingwg/multus-cni)) to
+prepare the pod for networking.
+The meta-plugin will invoke the CNI responsible for setting up the pod's default cluster network.
+After that, the meta-plugin iterates the list of secondary networks, invoking the corresponding CNI implementing
+the logic to attach the pod to that particular secondary-network. The CNI will use the details specified on
+the `network-attachment-definition` in order to do that.
 
 **NOTE**:
-- networks are **not** namespaced - i.e. creating multiple
-  `network-attachment-definition`s with different configurations pointing at the
-  same network (same `NetConf.Name` attribute) is **not** supported.
+
+- networks are **not** namespaced - i.e. creating multiple `network-attachment-definition`s with different
```

Comment:
maybe add another entry saying NAD updates are not supported ? i.e. the user must remove all workloads attached to the network (in all namespaces ...), then delete *all* NADs, then finally re-provision the "new" ones ? 

This could make sense on a separate `limitations` heading.

Reply:
I agree. Added to limitations.